classes/patreon_routing.php

```diff
@@ -4,15 +4,14 @@
 if ( ! defined( 'WPINC' ) ) {
 	die;
 }
-
+ 
```

Comment:
remove this rogue space

Reply:
Rogue space fix

https://github.com/codebard/patreon-wordpress/commit/120f3d40bb1020535b98494921e7d2c143a59455